.github/workflows/coveralls.yml

```diff
@@ -1,12 +1,11 @@
 name: Code coverage
-
 on:
+    workflow_dispatch:
```

Comment:
This action is still being run against pull_requests against the main branch but it's being called instead from the development entry-point.